backend/src/services/user-follower/user-follower.service.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  UserFollower as TUserFollower,
+  UserFollowerPayload,
+} from '~/common/types/types';
+import { ErrorMessage, HttpCode } from '~/common/enums/enums';
+import { HttpError } from '~/exceptions/exceptions';
+import { userFollower as userFollowerRep } from '~/data/repositories/repositories';
+
+type Constructor = {
+  userFollowerRepository: typeof userFollowerRep;
+};
+
+class UserFollower {
+  #userFollowerRepository: typeof userFollowerRep;
+
+  constructor({ userFollowerRepository }: Constructor) {
+    this.#userFollowerRepository = userFollowerRepository;
+  }
+
+  public getCountByUserId(userId: number): Promise<number> {
+    return this.#userFollowerRepository.getCountByUserId(userId);
+  }
+
+  public isFollowed(payload: UserFollowerPayload): Promise<TUserFollower> {
```

Comment:
```suggestion
  public checkIsFollowed(payload: UserFollowerPayload): Promise<TUserFollower> {
```
functions with this naming should return boolean